build-tools/packages/build-cli/src/commands/merge.ts

```diff
@@ -0,0 +1,194 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { getResolvedFluidRoot, GitRepo } from "@fluidframework/build-tools";
+import { Octokit } from "@octokit/core";
+import { Flags } from "@oclif/core";
+import { BaseCommand } from "../base";
+
+const owner = "microsoft";
+const repo = "FluidFramework";
+
+async function prExists(token: string, title: string): Promise<boolean> {
+    const octokit = new Octokit({ auth: token });
+    const response = await octokit.request("GET /repos/{owner}/{repo}/pulls", { owner, repo });
+
+    for (const data of response.data) {
```

Comment:
Just a small nit, might not even be nessesary but thought of a cool one liner for lines 17-23; 
`return response.data.some(data => data.title === title)`